project/CrossArchitecture.scala

```diff
@@ -0,0 +1,69 @@
+import sbt._
+import Keys.{name, unmanagedResourceDirectories, baseDirectory}
+import sbtcrossproject._
+import scala.scalanative.build
+import build.{Bits, ThirtyTwo, SixtyFour, TargetArchitecture}
+import scala.scalanative.sbtplugin.ScalaNativeCrossVersion
+import scala.scalanative.sbtplugin.ScalaNativePlugin
+import scala.scalanative.sbtplugin.ScalaNativePlugin.autoImport._
+
+case class CrossBitsPlatform(bits: Bits, lib: Boolean = false)
+    extends Platform {
+  def identifier: String = "_bits-" + bits.toString
```

Comment:
People often use `x32` and `x64` to distinguish two types of architectures. It's more concise and just as clear.